### DISCUSSION TITLE
What is M3.

### DISCUSSION THREAD

**Discussion by 1-Mythical** [winstxnhdw/lc-hax#23] [Q&A] [answered]
Also sounds quite stupid to ask. I thought this was some type of keybind. I also checked if this was Mouse 3, but I didn't see that do anything either.

**Comment by winstxnhdw** [comment, marked as answer]
It's `Mouse 3`. Your middle click. Read [this](https://github.com/winstxnhdw/lc-hax#triggermod) to understand what it does.

**Comment by 1-Mythical** [comment]
I looked at some of that. I think I was supposed to be looking at it, but randomly stuff would happen without me looking at it.

**Comment by winstxnhdw** [comment]
You’ll have to elaborate.

**Comment by 1-Mythical** [comment]
I was at the Company Building and when I pressed ```Mouse 3```, Jeb attacked and I didn't look at him.

**Comment by winstxnhdw** [comment]
Jeb’s collider spans the entire map, so doing `M3` anywhere would trigger him.
